src/bridge/ExposureCheck.ts

```diff
@@ -0,0 +1,11 @@
+import {NativeModules} from 'react-native';
+
+import {NotificationPayload, PeriodicWorkPayload} from './PushNotification';
+
+const ExposureCheckBridgeBare = NativeModules.ExposureCheck as {
+  scheduleExposureCheck(payload: PeriodicWorkPayload): Promise<any>;
```

Comment:
We'll need to add these for the iOS bridge to ensure it compiles before we merge this.